lib/cached_booking_locations_api.rb

```diff
@@ -1,11 +1,11 @@
-class CachedBookingLocationsApi < BookingLocations::Api
+class CachedBookingLocationsApi
   def initialize(cache: Rails.cache)
     @cache = cache
   end
 
   def find(location_id)
     cache.fetch(location_id, expires_in: 2.hours) do
```

Comment:
If is worth making the key more specific so it can't conflict if we use other caches later:
```
cache.fetch("Location: #{location_id}", expires_in: 2.hours) do
```